src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -51,41 +262,951 @@
+    interconnectClient.close();
+    licenseClient.deleteLicense(LICENSE_NAME);
+    licenseClient.close();
+    machineTypeClient.close();
+    networkEndpointGroupClient.deleteNetworkEndpointGroup(NETWORK_ENDPOINT_GROUP_NAME);
+    networkEndpointGroupClient.close();
+    networkClient.close();
+    nodeTemplateClient.close();
+    nodeTypeClient.close();
+    nodeGroupClient.close();
+    projectClient.close();
+    regionAutoscalerClient.close();
+    regionClient.close();
+    regionCommitmentClient.close();
+    regionDiskClient.close();
+    regionBackendServiceClient.close();
+    targetHttpProxyClient.close();
+    targetHttpsProxyClient.close();
+    targetInstanceClient.close();
+    Thread.sleep(30000);
+    targetPoolClient.deleteTargetPool(REGION_TARGET_POOL_NAME);
+    targetPoolClient.close();
+    targetSslProxyClient.close();
+    targetTcpProxyClient.close();
+    targetVpnGatewayClient.close();
+    urlMapClient.close();
+    vpnGatewayClient.close();
+    vpnTunnelClient.close();
+    zoneClient.close();
+    if (storage != null) {
+      boolean wasDeleted = RemoteStorageHelper.forceDelete(storage, BUCKET, 10, TimeUnit.SECONDS);
+      if (!wasDeleted && LOG.isLoggable(Level.WARNING)) {
+        LOG.log(Level.WARNING, "Deletion of bucket {0} timed out, bucket is not empty", BUCKET);
+      }
+    }
+  }
+
+  @Test
+  public void listAcceleratorTypesTest() {
+    AcceleratorTypeClient.ListAcceleratorTypesPagedResponse listAcceleratorTypes =
+        acceleratorTypeClient.listAcceleratorTypes(PROJECT_ZONE_NAME);
+    List<AcceleratorType> acceleratorTypes = Lists.newArrayList(listAcceleratorTypes.iterateAll());
+    assertThat(acceleratorTypes).isNotNull();
+    assertTrue(acceleratorTypes.iterator().hasNext());
+    assertNotNull(acceleratorTypes.iterator().next());
+  }
+
+  @Test
+  public void aggregatedListAcceleratorTypesTest() {
+    AggregatedListAcceleratorTypesPagedResponse aggregatedListAcceleratorTypes =
+        acceleratorTypeClient.aggregatedListAcceleratorTypes(PROJECT_NAME);
+    List<AcceleratorTypesScopedList> typesScopedLists =
+        Lists.newArrayList(aggregatedListAcceleratorTypes.iterateAll());
+    assertThat(typesScopedLists).isNotNull();
+    assertTrue(typesScopedLists.iterator().hasNext());
+    assertNotNull(typesScopedLists.iterator().next());
+  }
+
+  @Test
+  public void getAddressesTest() {
+    ProjectRegionAddressName projectRegionAddressName =
+        ProjectRegionAddressName.of(ADDRESS_NAME, DEFAULT_PROJECT, REGION);
+    Address address = addressClient.getAddress(projectRegionAddressName);
+    assertThat(address).isNotNull();
+    assertEquals(ADDRESS_TYPE, address.getAddressType());
+    assertEquals(ADDRESS_NAME, address.getName());
+    assertEquals(NETWORK_TIER, address.getNetworkTier());
+    assertEquals(REGION_LINK, address.getRegion());
+    assertEquals(ADDRESS_OPERATION_STATUS, address.getStatus());
+    assertEquals(ADDRESS_SELF_LINK, address.getSelfLink());
```

Comment:
this would be 
`assertThat(address.getAddressType()).isEqualsTo(ADDRESS_TYPE);`

Reply:
> this would be
> `assertThat(address.getAddressType()).isEqualsTo(ADDRESS_TYPE);`

I think both are same so no need to used that one.

Reply:
Please refer above comment on Truth

Reply:
done